python/pyproject.toml

```diff
@@ -32,4 +32,7 @@ requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.scripts]
-serve = "arflow.main:serve"
+serve = "arflow.serve:serve"
+docs = "arflow.docs:make_docs"
+simple-server = "examples.simple_server:main"
+simple-replay = "examples.simple_replay:main"
```

Comment:
I don't think centralizing all bash script code to the pyproject.toml file is a good practice. The Python side code will be published to pypi as both a package and a CLI tool. The command defined in the pyproject.toml file will be exposed as CLI subcommand, for example: arflow serve. The scripts for executing example code shouldn't be added to there.

Reply:
I see. Will revert this